cjworkbench/tests/test_catalogs.py

```diff
@@ -0,0 +1,848 @@
+from django.test import SimpleTestCase
+from babel.messages.catalog import Catalog, Message
+from cjworkbench.i18n.catalogs.util import (
+    message_unique_identifier,
+    find_corresponding_message,
+    find_corresponding_string,
+    find_fuzzy_messages,
+    fill_catalog,
+    mark_fuzzy,
+    remove_strings,
+)
+from cjworkbench.i18n.catalogs.merge import _merge_source_catalog, _merge_catalog
+
+
+class CatalogTest(SimpleTestCase):
```

Comment:
It doesn't need to, but I like `self.assert...` more than `assert_...`

Reply:
I'd prefer `assert_...()`, please. We already use that pattern with Numpy, Pandas, Arrow, ProcessResult.

Someday, with any luck, we'll abandon the `unittest` module altogether. All the cool kids are using pytest, because it's better in every way.